README.md

```diff
@@ -1,6 +1,14 @@
 # ODPC
```

Comment:
@mstokericatt denk je dat dit duidelijk genoeg is?

Reply:
yep